File: JavaScript/SortColors/SortColors.js
/**
 * (118ms)
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
	var numOfZero = 0, numOfOne = 0, numOfTwo = 0;
    for(var num of nums) {
    	if(num == 0) {
    		numOfZero++;
    	} else if(num == 1) {
    		numOfOne++;
    	} else {
    		numOfTwo++;
    	}
    }
    for(var i = 0; i < numOfZero; i++) {
    	nums[i] = 0;
    }
    for(var i = 0; i < numOfOne; i++) {
    	nums[i + numOfZero] = 1;
    }
    for(var i = 0; i < numOfZero; i++) {
    	nums[i + numOfZero + numOfOne] = 2;
    }
};

File: Python/SortColors/SortColors.py
import collections


class Solution(object):
	# (45ms)
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        dt = collections.Counter(nums)
        for i in xrange(dt[0]):
        	nums[i] = 0
        for i in xrange(dt[1]):
        	nums[i + dt[0]] = 1
        for i in xrange(dt[2]):
        	nums[i + dt[0] + dt[1]] = 2

nums = [1, 0]
Solution().sortColors(nums)
print nums